components/chromatin-biosample-facets.js

```diff
@@ -24,24 +24,26 @@ export default function ChromatinBiosampleFacets({
         <div>Filter by biosample </div>
       </div>
       <div className="text-data-label text-sm italic">Grouped by organs</div>
-      <div className="overflow-y-auto h-80 border-2 border-panel">
+      <div className="overflow-y-auto h-80 border-2 border-panel p-1">
         {facets.map((facet) => {
           const key = facet.biosample.replace(/[^\w\s]/gi, "").toLowerCase();
           const isSelected = biosampleFilters.includes(facet.biosample);
           return (
             <div key={key}>
-              <div className="grid grid-cols-4 ">
+              <div className="grid grid-cols-4">
                 <div className="col-span-3">
                   <button
-                    className={` flex text-sm hover:bg-highlight gap-1
+                    className={` text-sm hover:bg-highlight
                     ${isSelected && "border-solid border-2 border-brand"}`}
```

Comment:
Not part of this branch’s changes, but if `isSelected` is false, do you get a class of `false` inserted in this `className` string?Also, there’s a space before `text-sm` in the class string. Doesn’t hurt anything, but not sure the reason.

Reply:
Yup, that works!